R/FilteredDataCDISC.R

```diff
@@ -105,30 +93,44 @@ CDISCFilteredData <- R6::R6Class( # nolint
           )
         )
 
+        join_keys <- self$get_join_keys()
+        keys <-
+          if (!is.null(join_keys)) {
+            join_keys$get(parent_dataname, dataname)
+          } else {
+            character(0)
+          }
+        parent_keys <- names(keys)
+        dataset_keys <- unname(keys)
+
+        y_arg <-
+          if (length(parent_keys) == 0L) {
+            parent_dataname
+          } else {
+            sprintf("%s[, c(%s), drop = FALSE]", parent_dataname, toString(sprintf("\"%s\"", parent_keys)))
```

Comment:
please use dQuote or sQuote
```suggestion
            sprintf("%s[, c(%s), drop = FALSE]", parent_dataname, toString(dQuote(parent_keys)))
```

Reply:
:+1: